src/components/widgets/eve-mail/eve-mail.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {eveMailFetchHeaders} from '../../../actions/eve-mail';
+import axios from 'axios';
+const EVE_PIC = require('../../../assets/eve-login.png');
+
+class EveMail extends Component {
+  constructor(props, context){
+    super(props, context);
+    this.state = {
+      screen: null
+    };
+  }
+  handleClick(){
```

Comment:
missing whitespace between closing parenthesis and opening curly bracket: https://github.com/airbnb/javascript#whitespace--before-blocks